OpenRA.Game/Server/Server.cs

```diff
@@ -78,6 +78,8 @@ public sealed class Server
 		readonly List<GameInformation.Player> worldPlayers = new List<GameInformation.Player>();
 		readonly Stopwatch pingUpdated = Stopwatch.StartNew();
 
+		readonly Dictionary<int, List<long>> playerMessageTracker = new Dictionary<int, List<long>>();
```

Comment:
Imo we should move this to a dedicated class, called `PlayerMessageTracker ` which should host the logic down at line 1000+, but let the dispatch stuff still be in the server class.

Reply:
Added a PlayerMessageTracker class